lib/saml_idp/xml_security.rb

```diff
@@ -201,7 +201,11 @@ def validate_doc_embedded_signature(base64_cert, soft = true)
           canon_hashed_element          = hashed_element.canonicalize(canon_algorithm,
                                                                       inclusive_namespaces)
 
-          digest_algorithm              = algorithm(REXML::XPath.first(ref, '//ds:DigestMethod'))
+          digest_algorithm              = algorithm(REXML::XPath.first(
```

Comment:
is it possible to write a spec for this to show it being missed w/o namespace?

Reply:
i was just about to write a comment saying "i will write a spec after standup"! i just wanted to get it in front of people's eyes to make sure i wasn't doing something wild

Reply:
also the spacing is silly, we should probably remove the horizontal alignment if we are going to have new lines like this
```suggestion
          digest_algorithm = algorithm(REXML::XPath.first(
```

Reply:
so yes, i think it is!

Reply:
the spacing is silly but the whole file is like that, i'd rather do it in a separate PR so as not to make this one too messy

Reply:
linting fixes here! https://github.com/18F/saml_idp/pull/103